tests/gtest/aviftest_helpers.cc

```diff
@@ -340,9 +341,15 @@ AvifImagePtr ReadImage(const char* folder_path, const char* file_name,
 }
 
 bool WriteImage(const avifImage* image, const char* file_path) {
-  return avifPNGWrite(file_path, image, /*requestedDepth=*/0,
-                      AVIF_CHROMA_UPSAMPLING_BEST_QUALITY,
-                      /*compressionLevel=*/0);
+  if (!image || !file_path) return false;
+  const size_t str_len = std::strlen(file_path);
```

Comment:
std::string is already used in the file, but it's probably just as much work to use it here instead of cstring.

Reply:
If there was `std::string::ends_with()` I would reconsider but this is C++20 only.